administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -6,15 +6,37 @@ import FlipMove from 'react-flip-move'
 import { useTranslation } from 'react-i18next'
 import styled from 'styled-components'
 
-import { GetApplicationsQuery } from '../../generated/graphql'
 import NonIdealState from '../../mui-modules/NonIdealState'
 import StandaloneCenter from '../StandaloneCenter'
 import ApplicationCard from './ApplicationCard'
 import type { ApplicationCardProps } from './ApplicationCard'
 import ApplicationStatusBar from './ApplicationStatusBar'
-import { ApplicationStatusBarItemType, barItems } from './constants'
 import usePrintApplication from './hooks/usePrintApplication'
-import { getApplicationStatus, getVerificationStatus } from './utils'
+import { ApplicationStatusBarItemType, ApplicationVerificationStatus, GetApplicationsType } from './types'
+import { applicationEffectiveStatus } from './utils'
+
+export const barItems: ApplicationStatusBarItemType[] = [
+  {
+    title: 'allApplications',
+    status: undefined,
+  },
+  {
+    title: 'accepted',
+    status: ApplicationVerificationStatus.Approved,
+  },
+  {
+    title: 'rejected',
+    status: ApplicationVerificationStatus.Rejected,
+  },
+  {
+    title: 'withdrawed',
```

Comment:
```suggestion
    title: 'withdrawn',
```
or revoked as an option

Reply:
There were a lot of this specific typo in the code base, fixed them.